src/app/Scenes/AuctionResult/AuctionResult.tsx

```diff
@@ -54,9 +54,13 @@ export const AuctionResult: React.FC<Props> = (props) => {
 
   const tracking = useTracking()
 
+  const headerTitleString = auctionResult.dateText
```

Comment:
very minor: IMO, there is no need to add the variable type to the variable name.
```suggestion
  const headerTitle = auctionResult.dateText
```